pkg/utils/dyn_pprof_unix.go

```diff
@@ -0,0 +1,40 @@
+// +build linux darwin freebsd unix
+// Copyright 2020 PingCAP, Inc. Licensed under Apache-2.0.
+
+package utils
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+)
+
+const (
+	startPProfSignal = syscall.SIGUSR1
+)
+
+var (
+	signalChan = make(chan os.Signal, 1)
+)
+
+// StartDynamicPProfListener starts the listener that will enable pprof when received `startPProfSignal`.
+func StartDynamicPProfListener() {
+	signal.Notify(signalChan, startPProfSignal)
+	go onSignalStartPProf(signalChan)
+	log.Info(fmt.Sprintf("dynamic pprof started, you can enable pprof by `kill -s %d %d`)",
+		startPProfSignal,
+		os.Getpid()))
```

Comment:
IMO we don't need to advertise this trick to all users.

```suggestion
```

Reply:
Well, how about change it to DEBUG level?

Reply:
Debug is OK but still not useful. I don't see any scenario where you can read the log and perform `kill -USR1 $PID` but cannot `ps aux` to get the PID.